CHANGES.md

```diff
@@ -17,7 +17,7 @@ aligning related proposals, partly motivating a new Screens interface.
   - User-friendly screen labels, which vastly improve custom multi-screen picker and configuration UIs for apps
   - Screen change events, including changes to the multi-screen bit, to obviate polling
 - Chrome Security requested that this feature be disabled by default on embedded pages via a [permissions policy](https://w3c.github.io/webappsec-permissions-policy/)
-  - In the second origin trial, cross origin iframes need to specify an `allow="window-placement"` attribute.
+  - In the second origin trial, cross origin iframes need to specify an `allow="window-management"` attribute.
```

Comment:
Let's actually document the renaming in this file.

Reply:
Added an entry mentioning the rename.